.travis.yml

```diff
@@ -12,7 +17,7 @@ before_script:
 script:
 - |
   # get ourselves a MIR-ful libstd
-  xargo/build.sh
+  cd xargo && RUSTFLAGS='-Zalways-encode-mir' xargo build && cd ..
```

Comment:
Okay, turns out it supports no flags. So, it's not helpful. I suggest to just use dirname.